### DISCUSSION TITLE
Investigate if we could use https://freemathapp.org/

### DISCUSSION THREAD

**Discussion by zachberry** [ucfopen/Obojobo#1693] [Ideas]
Could be useful for us: https://freemathapp.org/

**Comment by maufcost** [comment]
Do you have any features specifically that you would like to integrate from them on Obojobo? Or just access to perhaps freemathapp's API or package (if allowed)? If you would like to, I could email their dev team with some questions.

**Comment by jaltekruse** [comment]
Hello! Creator of Free Math here, I would be happy to meet to discuss potential integration between our platforms. Free Math is GPL, and I have considered re-licensing to AGPL, so I would be open to that if is necessary.

Feel free to book some time with me using this link - https://calendly.com/freemath
